fiftyone/operators/__init__.py

```
@@ -19,6 +19,7 @@
 )
 from .utils import ProgressHandler, is_new
 from .panel import Panel, PanelConfig
+from .store import ExecutionStore
```

Ruff

`.store.ExecutionStore` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)